modules/bcftools/convert/main.nf

```diff
@@ -0,0 +1,41 @@
+process BCFTOOLS_CONVERT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::bcftools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bcftools:1.15.1--h0ea216a_0':
+        'quay.io/biocontainers/bcftools:1.15.1--h0ea216a_0' }"
+
+    input:
+    tuple val(meta), path(input), path(input_index), path(bed)
```

Comment:
Most `bcftools` modules use `vcf` and `tbi` as input name, perhaps it makes sense to keep this naming scheme?
Also, I'd pull the bed file from the input tuple

Reply:
The `input` in this case isn't always a `vcf` or `tbi`, so maybe `input` is the better name?

Reply:
good call!

Reply:
possible other inputs are `tsv`, `hap`, `sample`...